libs/common/mem_operations.hpp

```diff
@@ -27,21 +26,25 @@ namespace iroha {
 
 #ifdef __linux__
   inline uint64_t getMemoryUsage() {
-    auto parseLine = [](char *line) {
-      while (*line >= '0' && *line <= '9') ++line;
+    auto parseLine = [](char const *line) {
+      while (!std::isdigit(*line)) ++line;
       return (uint64_t)atoll(line);
     };
 
     uint64_t result = 0ull;
     char line[128];
 
-    FILE *file = fopen("/proc/self/status", "r");
-    while (fgets(line, 128, file) != NULL)
-      if (strncmp(line, "VmSize:", 7) == 0) {
-        result = parseLine(line + 7);
+    std::unique_ptr<FILE, int (*)(FILE *)> file(fopen("/proc/self/status", "r"),
+                                                &fclose);
+
+#define VM_SZ_FIELD "VmSize:"
```

Comment:
why `#define` instead of `constexpr`:
```
constexpr char VM_SZ_FIELD[] = "VmSize:";
```
it would be more C++ way:)

Reply:
ok)